lib/ProductOpener/Test.pm

```diff
@@ -266,7 +266,7 @@ sub ensure_expected_results_dir ($expected_results_dir, $update_expected_results
 	return 1;
 }
 
-=head2 compare_to_expected_results($object_ref, $expected_results_file, $update_expected_results) {
+=head2 compare_to_expected_results($object_ref, $expected_results_file, $update_expected_results, $test_ref = undef) {
```

Comment:
$test_ref is currently not used

Reply:
it will be used when I merge data quality :-D